ern-container-gen-ios/src/IosGenerator.ts

```diff
@@ -82,6 +82,10 @@ export default class IosGenerator implements ContainerGenerator {
   public async fillContainerHull(
     config: ContainerGeneratorConfig
   ): Promise<void> {
+    if (config.targetPlatform === 'ios' && process.platform !== 'darwin') {
```

Comment:
No need for `config.targetPlatform` check here. This is the iOS container generator implementation so it's explicit that the target platform is iOS ;)

Reply:
Haha yeah of course, makes sense :+1: